srcpkgs/wezterm/wezterm-terminfo.INSTALL

```diff
@@ -0,0 +1,6 @@
+case "$ACTION" in
+post)
+	# Compile the terminfo description.
+	tic -xs usr/share/terminfo/w/wezterm.terminfo
```

Comment:
this should be done in post_install instead of at install-time (was changed in #38179)

Reply:
I am not sure, how?

Reply:
I gave a go at this; in an updated commit....

Reply:
you should only need

```bash
do_install() {
	# existing stuff ...
	vmkdir usr/share/terminfo
	tic -sx -o ${DESTDIR}/usr/share/terminfo termwiz/data/wezterm.terminfo
}

wezterm-terminfo_package() {
	# ...
	pkg_install() {
		vmove usr/share/terminfo
	}
}

Reply:
Thanks!